### DISCUSSION TITLE
Copy settings

### DISCUSSION THREAD

**Discussion by RandomKuchen** [slyh/Cities2-TrafficLightsEnhancement#86] [Ideas]
Hey, 

it would be nice if you can copy settings from one intersection to another. I would make it easy to use it, when your city is already quite large. 

![grafik](https://github.com/slyh/Cities2-TrafficLightsEnhancement/assets/63271030/0d72b5c9-8f80-4819-a462-4b99526bef5e)

And Ive just updated your German translations on Crowdin. :)

**Comment by slyh** [comment]
Thanks for your contribution. I will incorporate all the translation updates in the next update.

Regarding the config copy and paste feature, I believe MioHotkeyMod has it built-in. You might want to give it a try, as this feature may not be implemented soon.
________________________________
From: Mark Hennies ***@***.***>
Sent: Saturday, March 16, 2024 11:58:58 PM
To: slyh/Cities2-TrafficLightsEnhancement ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [slyh/Cities2-TrafficLightsEnhancement] Copy settings (Discussion #86)


Hey,

it would be nice if you can copy settings from one intersection to another. I would make it easy to use it, when your city is already quite large.

grafik.png (view on web)<https://github.com/slyh/Cities2-TrafficLightsEnhancement/assets/63271030/0d72b5c9-8f80-4819-a462-4b99526bef5e>

And Ive just updated your German translations on Crowdin. :)
